docs/configuration.md

```diff
@@ -189,6 +194,15 @@ matchers:
 mute_time_intervals:
   [ - <string> ...]
 
+# Times when the route should be active. These must match the name of a
+# mute time interval defined in the mute_time_intervals section. 
+# Additionally, the root node cannot have any active times.
+# Only when a route is active it will send any notifications, but
+# otherwise acts as muted (it ends the route-matching process
+# if the `continue` option is not set.)
```

Comment:
```suggestion
# The route will send notifications only when active, but otherwise
# otherwise acts normally (including ending the route-matching process
# if the `continue` option is not set).
```